### PR TITLE
[Cognitive Services] Update endpoint template for Content Moderator SDK.

### PR DESCRIPTION
**Pull request by yangyuan**
https://github.com/Azure/azure-rest-api-specs/pull/3505
1. Update auto generated code.
2. Update tests accordingly.
3. Bump version/Add AssemblyInfo/Update release notes.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.ContentModerator</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Content Moderator APIs.</Description>
-    <Version>0.12.1-preview</Version>
+    <Version>0.13.0-preview</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.ContentModerator</AssemblyName>
     <PackageTags>ContentModerator;Content Moderator;</PackageTags>
     <PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj]: Please update the `PackageReleaseNotes`

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Cognitive Services Content Moderator SDK")]
+[assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services Content Moderator APIs.")]
+
+[assembly: AssemblyVersion("0.13.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Properties/AssemblyInfo.cs]: `AssemblyVersion` must be set to the first nuget package version that was published.

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Please fix the build errors too

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj (hunk 1 above)
Comment (yangyuan) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj]: Updated

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (yangyuan) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Properties/AssemblyInfo.cs]: Checked the first was 0.9.0, so changed AssemblyVersion to 0.9.0.0

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Comment by yangyuan** [comment]
@dsgouda I guess the building error haven't been fixed yet. I'm looking into it.
It's weird because I didn't change test proj config or test logic

**Comment by shahabhijeet** [comment]
@yangyuan you have a test failure.
Open this file in VS and you will see the test failure.
https://azuresdkci.westus2.cloudapp.azure.com/job/CI-NetSDK_U/226/artifact/TestResults/net452/

**Comment by yangyuan** [comment]
Thanks for the hint, it seems I might need to do some changes in the API specifications to fix it.

**Comment by yangyuan** [comment]
Sent a PR for swagger which should be helpful in fixing this issue,
https://github.com/Azure/azure-rest-api-specs/pull/3681

**Comment by yangyuan** [comment]
@dsgouda looks like the issue get fixed.

**Comment by swiftarrow11** [comment]
Looks good. @sanjeev3, we should plan to update our SDK samples to use this version. Note: Our sdk will now use the Endpoint parameter instead of baseurl

**Pull request merged**